_posts/2017-06-28-Comparing_apis_in_go_node_and_dotnet_core.md

```diff
@@ -149,16 +149,59 @@ func main() {
 
 I was the least happy with how this code turned out. The serialisation stuff was difficult to do, especially in comparison to node. I also find the code to be much less succinct than either of the other languages. I expect this could be alleviated if I knew a few more libraries to help with these things. Its also important to note that this was my very first go program, whereas I have quite a bit of experience with both c# and client-side javascript.
 
+## Rust (thanks[@Kazetsukai](https://github.com/lukemcgregor/basic-api/pull/2))
+
+I know absolutally nothing about rust (before today), so fun times. Rust is a functional language designed for performance.
```

Comment:
I don't believe Rust identifies as specifically a functional language? It certainly has some functional looking features but I believe it is primarily imperative?